### DISCUSSION TITLE
Using Regional Prompter in conjunction with ControlNet

### DISCUSSION THREAD

**Discussion by dayunbao** [hako-mikan/sd-webui-regional-prompter#189] [Q&A] [answered]
I just learned about this, and am quite excited to try it out.  どうもありがとうございます, hako-mikanさん!  

I'm building a comic book creation app that will make heavy use of ControlNet's OpenPose preprocessor, and was wondering if it's possible to use that in conjunction with Regional Prompter?  For example, if I feed ControlNet/OpenPose an image with two figures fighting, and I want those two figures to appear in a certain part of the final image, can I use Regional Prompter to place the ControlNet/OpenPose output in the center of the image?  I saw that someone asked a similar question, but it was never answered.

Thanks in advance for any help!

**Comment by hako-mikan** [comment]
Maybe you can.
Using the Images generated by openpose editor, I could generate without problems.
![test](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/122196982/817e7ae0-4773-402e-90b0-fc443c78ca54)
![test2](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/122196982/46c0f8da-7e07-471a-be8f-4ed53dce6200)

**Reply by dayunbao** [reply]
So that image was generated with both an active ControlNet (using OpenPose) and Regional Prompter active with different prompts for different parts of the image?

**Comment by hako-mikan** [comment, marked as answer]
Oh, my example was misleading, wasn't it?
You're right, I'm using separate prompts.
![test3](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/122196982/f11b68ba-021f-4f80-825e-c01b441a9a59)
![test2](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/122196982/e111706e-7693-4b05-916e-90ecbabf1106)

**Reply by dayunbao** [reply]
That's amazing!  Thanks!  

I'm going to be building a custom UI for Regional Prompter in my app, and using it via the API.  I'm quite excited to combine this with some of the other amazing tools people have made, along with some of my own.  I think the results will be really good.

Sorry for the late reply!
